### PR TITLE
feat: add react hook form to login page

### PR DESCRIPTION
**Pull request by dephraiim**

### CODE REVIEW

File: src/components/onboarding/company.tsx [hunk 1]
```diff
@@ -1,10 +1,84 @@
...
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form"
+
+const onboardingSchema = z.object({
+  firstName: z.string().min(1, {
```
Comment (dahal) [inline src/components/onboarding/company.tsx:20]: Lets design the schema in this way 👇🏼 , separating user and company
```
user: {
  firstName: z.string().min(1, {
    message: "First name is required",
  }),
  ...
},

company: {
  name: z.string().min(1, {
    message: "Company name is required",
  }),
  ...,
}
```

Also, lets design db schema as accordingly, change `org => company`

**Review by dahal — CHANGES_REQUESTED** [review]
Incredible work so far 👏 . While at it, do you want work on #10 as well, i will assign that to you if you want.

File: src/components/onboarding/company.tsx (hunk 1 above)
Comment (dephraiim) [reply src/components/onboarding/company.tsx]: 👍

**Review by tedspare — APPROVED** [review]
LGTM, colossal effort Duncan 🚢 🐐

**Review by dahal — APPROVED** [review]
🚀🚀🚀

**Pull request merged**